### ISSUE TITLE
Staying as Single Celled Organism when advancing to Multicellular stage

### ISSUE THREAD

**Issue by AmcYeet** [Revolutionary-Games/Thrive#3371]
when I advance to multicellular stage only the other one of my species it spawns beside me  actually has more than one cell, I do not.

**Comment by Oliveriver** [comment]
This is intentional. As you collect resources, you will grow the other cells.

There has been some internal discussion about how intuitive this system is, since it can lead to this sort of confusion. But as this is a prototype, such changes are not a priority.

**Comment by hhyyrylainen** [comment]
Yeah, once that is no longer a prototype we can add a tutorial popup like "Budding reproduction works like x" first time when exiting the multicellular editor.
